fix #276392: shift+click to select in foto mode doesn't work

jthistle opened the pull request:
Resolves: https://musescore.org/en/node/276392
Ready to merge.

mattmcclinch commented on mscore/events.cpp:
```diff
@@ -261,6 +261,7 @@ void ScoreView::mouseReleaseEvent(QMouseEvent*)
             case ViewState::ENTRY_PLAY:
             case ViewState::FOTO:
```
There needs to be a `break;` before `case ViewState::FOTO_LASSO:`

mattmcclinch commented on mscore/events.cpp:
```diff
@@ -879,6 +889,7 @@ void ScoreView::changeState(ViewState s)
                   break;
             case ViewState::ENTRY_PLAY:
```
There needs to be a `break;` here.

jthistle replied:
Thanks for the review. Since I'm relatively new to C++, could you explain why this is?

mattmcclinch replied:
C++ uses the concept of "fall through" to handle multiple cases in a switch statement. If you do not want the handling of one case to fall through to the next one, you must use a `break;` to exit the switch statement. Since originally there was no code to be executed for any of these cases, there was no reason to insert a `break;` between each of them.

jthistle replied:
Great, that makes sense, thanks.

jthistle commented:
@mattmcclinch @anatoly-os Changes made, now ready to merge I think

jthistle commented:
The code for the Foto Lasso was already written in 3.0, I just implemented the function calls and fixed a small bug with it. I tested it and it worked as expected, so I don't think it should need rewriting from scratch.
